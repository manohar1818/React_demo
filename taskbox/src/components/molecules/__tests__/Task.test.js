import React from "react";
import { shallow } from "enzyme";
import Task from "../Task/Task";

describe("Testing the task component", () => {
  const taskProps = {
    endDate: new Date("10-06-2020").getFullYear(),
    description: "Testing",
  };
  let wrapper;
  beforeEach(()=>{
     wrapper = shallow(<Task {...taskProps} />);

  })
  it("Test the checkbox Component in task Component", () => {
    
    expect(wrapper.find("CheckBoxComponent").props().size).toEqual("small");
    expect(wrapper).toMatchSnapshot();
  });
  it("Test the typoraphy Component in task Component", () => {
    expect(wrapper.find('#description').props().children).toEqual("Testing");
  });
  it("Test the Chip Component in task Component", () => {
    expect(wrapper.find('ChipComponent').props().label).toEqual( "Jan 1,  5:30 AM");
  });
});

