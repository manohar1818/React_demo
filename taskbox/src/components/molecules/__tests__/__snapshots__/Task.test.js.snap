// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the task component Test the checkbox Component in task Component 1`] = `ShallowWrapper {}`;
